### PR TITLE
feat(snippet): add `Make custom apps icons thicker`

### PR DESCRIPTION
**Pull request by Plueres**
also added image for the snippet

This changes the custom apps icons to have a thicker border just like the other icons in the list where custom apps reside

### CODE REVIEW

**Comment by Shub3am** [comment]
Looks good to me

File: packages/marketplace/src/resources/snippets.ts [hunk 1]
```diff
@@ -179,4 +179,10 @@ export default [
     "code": "section.main-shelf-shelf[aria-label='Recently played'] { visibility: hidden; position: absolute; }",
     "preview": "resources/assets/snippets/remove-recently-played.png",
   },
+  {
+    "title": "Modernize custom app icons",
+    "description": "Gives icons from the sticky list a thicker border",
```
Comment (ririxidev) [inline packages/marketplace/src/resources/snippets.ts:184]: thicker != more modern

```suggestion
    "title": "Make custom app icons more thick",
    "description": "Gives icons from the sticky list a thicker border",
```

such, edit preview file name as well

**Review by ririxidev — DISMISSED** [review]

File: packages/marketplace/src/resources/snippets.ts (hunk 1 above)
Comment (Plueres) [reply packages/marketplace/src/resources/snippets.ts:184]: yeah that's a much better name. I'll also change the file name in the next commit

**Review by kyrie25 — COMMENTED** [review]
LGTM

**Review by kyrie25 — APPROVED** [review]

**Pull request merged**